docker/Dockerfile

```diff
@@ -162,6 +162,7 @@ FROM dependencies-dev-platform-$TARGETARCH as dependencies-dev
 RUN npm install --global markdownlint-cli@0.31.1
 
 # Install development-specific dependencies of Nautobot
+# hadolint ignore=DL3059
```

Comment:
Note to other reviewers: it looks like hadolint 2.10 is reporting an issue not previously reported by hadolint 2.0 - this has not been failing in CI because our CI action is pinned to `hadolint/hadolint-action@1.6.0` which uses hadolint 2.7. (Confusing? Yes.) We need to fix CI at some point so that it's running the intended version of hadolint.

Reply:
Fixed in #1687

Reply:
Thanks! I pulled in changes from develop, but seems to show that I am making changes to the dockerfile, if I need to recreate this PR, please let me know.